TestSwitchApi/TestSwitchApi/Models/DataModels/CandidateTestModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    namespace TestSwitchApi.Models.DataModels
+    {
+        public class CandidateTestModel
+        {
+            [Key]
+            [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+            public int Id { get; set; }
+            public int CandidateId { get; set; }
+            public int TestId { get; set; }
+            public string TestResult { get; set; }
+            public string TestAnswer { get; set; }
```

Comment:
Do we actually need the `Id` for this table?
Could the primary key be the combination of the CandidateId and the TestId?

That way the DB would automatically enforce that each candidate could only take each test once! :)

Reply:
primary key is now a combination of  CandidateId and the TestId